File: app/src/main/java/com/example/nijimac103/easyclient_mvvm/view/adapter/CustomBindingAdapter.java
package com.example.nijimac103.easyclient_mvvm.view.adapter;

import android.databinding.BindingAdapter;
import android.view.View;

public class CustomBindingAdapter {
    //xmlに定義する際のBindingAdapter
    @BindingAdapter("visibleGone")
    public static void showHide(View view, boolean show) {
        view.setVisibility(show ? View.VISIBLE : View.GONE);
    }
}

File: app/src/main/java/com/example/nijimac103/easyclient_mvvm/view/adapter/CustomBindingAdapter.kt
package com.example.nijimac103.easyclient_mvvm.view.adapter

import android.databinding.BindingAdapter
import android.view.View

object CustomBindingAdapter {
    //xmlに定義する際のBindingAdapter
    @JvmStatic
    @BindingAdapter("visibleGone")
    fun showHide(view: View, show: Boolean) {
        view.visibility = if (show) View.VISIBLE else View.GONE
    }
}
